### PR TITLE
fde.md:Modified fde guide for bootloader install, indicating install for UEFI systems.

### PR DESCRIPTION
**Pull request by Gobybear**
Hello,
My first ever contribution.
Modified the fde guide and added grub install command for UEFI systems.
I tested it on firefox and it seems to work form my machine, no additional testings made.
Have a nice day.

### CODE REVIEW

**Comment by Gobybear** [comment]
Ah, my PR message doesn't follow CONTRIBUTING guidelines.

**Comment by abenson** [comment]
PR Title and Commit Message are separate. On your branch, you'll need to `git commit --amend`, fix the message, and then `git push -f`.

File: src/installation/guides/fde.md [hunk 1]
```diff
@@ -274,6 +274,12 @@ Next, install the boot loader to the disk.
 [xchroot /mnt] # grub-install /dev/sda
 ```
 
+Or, for UEFI systems.
+
+```
+[xchroot /mnt] # grub-install --target=x86_64-efi --efi-directory=/efi
```
Comment (abenson) [inline src/installation/guides/fde.md:280]: Why are you forcing ESP to `/efi`?

Comment (Gobybear) [reply src/installation/guides/fde.md:280]: Doesn't work otherwise.
However I learned it from using gentoo, can't tell if grub would work without it.

Comment (classabbyamp) [reply src/installation/guides/fde.md:280]: this should probably match the [regular chroot install guide](https://docs.voidlinux.org/installation/guides/chroot.html#installing-grub)

Comment (Gobybear) [reply src/installation/guides/fde.md:280]: Will do

**Comment by Gobybear** [comment]
>  1 workflow awaiting approval 
Does that mean I need to wait for approval now ?
It's the first time I do this so I don't know.

File: src/installation/guides/fde.md (hunk 1 above)
Comment (Gobybear) [reply src/installation/guides/fde.md]: Deleted --efi-directory=/efi to match the chroot install guide.

**Pull request closed without merging**